apps/web/modules/ui/components/editor/components/toolbar-plugin.test.tsx

```
@@ -253,4 +292,118 @@ describe("ToolbarPlugin", () => {
+        <ToolbarPlugin
+          getText={() => "Sample text"}
+          setText={vi.fn()}
+          editable={true}
+          container={document.createElement("div")}
+        />
+      );
+
+      const boldButton = screen.getByTestId("bold-icon").parentElement;
+      expect(boldButton).toBeInTheDocument();
+
+      await userEvent.click(boldButton!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)